spring-integration-file/src/main/java/org/springframework/integration/file/config/FileParserUtils.java

```diff
@@ -118,7 +124,7 @@ static void configureFilter(BeanDefinitionBuilder synchronizerBuilder, Element e
 			}
 			if (count != 1) {
 				parserContext.getReaderContext().error("at most one of 'filename-pattern', " +
-						"'filename-regex', 'filter' or 'filter-expression' is allowed on remote file inbound adapter",
+								"'filename-regex', 'filter' or 'filter-expression' is allowed on remote file inbound adapter",
```

Comment:
we have the same issue in Spring Boot. @philwebb was working on a generic formatter solution but this had to be put on hold to work on other things. It's current ad-hoc at the moment and I don't think you can't configure STS and IJ to match perfectly (especially with indent, the algorithm doesn't match for everything).